ShchyrbaOleksandr/HW7/task2.py

```diff
@@ -0,0 +1,30 @@
+def area_rectangle():
```

Comment:
good job
look at this implementation

def return_square_rectangle():
    a = float(input("Input width: "))
    b = float(input("Input height: "))
    return  "Square of rectangle is = {}".format(a*b)
 
def return_square_triangle():
    a = float(input("Input basis: "))
    h = float(input("Input height: "))
    return "Square of triangle is = {}".format(0.5 * a * h)
 
def return_square_circle():
    PI = 3.14
    r = float(input("Input radius: "))
    return "Square of circle is = {}".format(PI * r**2)
 
figure = input("1-rectangle, 2-triangle, 3-circle: ")

if figure == '1':
	print(return_square_rectangle())
elif figure == '2':
	print(return_square_triangle())
elif figure == '3':
	print(return_square_circle())
else:
	print("Input error")